dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/src/org/apache/cxf/jaxws/ServiceImpl.java

```diff
@@ -193,6 +212,13 @@ private void initializePorts() {
                     } else if (e instanceof javax.wsdl.extensions.soap.SOAPBinding) {
                         bindingID = SOAPBinding.SOAP11HTTP_BINDING;
                     }
+                } else {
```

Comment:
The else statement with the trace (lines 215 to 219) may not be needed as we log Binding id anyway on line 221, if it's not set, it will be logged as null.